README.md

```diff
@@ -384,22 +322,50 @@ Use the `configlet` tool to generate a README from shared metadata:
-[porting an exercise from another language](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md),
-or [creating an automated test generator](#implementing-a-generator).
+If you're just getting started and looking for a helpful way to get involved, take a look at [regenerating the test suites][#regenerating-a-test-suite], [porting an exercise from another language][port exercise], or [creating an automated test generator][#implementing-a-generator].
 
 ## Ruby icon
+
 The Ruby icon is the Vienna.rb logo, and is used with permission. Thanks Floor
 Dress :)
 
+[#anatomy-of-an-exercise]: #anatomy-of-an-exercise
+[assertion]: https://github.com/exercism/ruby/blob/master/lib/generator/exercise_case/assertion.rb
+[#canonical-data]: #canonical-data
+[canonical data]: https://github.com/exercism/problem-specifications/tree/master/exercises
+[canonical exercises]: https://github.com/exercism/problem-specifications/tree/master/exercises
+[#changing-a-generated-exercise]: #changing-a-generated-exercise
+[configlet-workflow-badge]: https://github.com/exercism/ruby/workflows/Configlet%20CI/badge.svg
+[configlet-workflow]: https://github.com/exercism/ruby/actions?query=workflow%3A%22Configlet+CI%22+branch%3Amaster
+[configure the remote]: https://help.github.com/articles/configuring-a-remote-for-a-fork/
+[#contributing-guide]: #contributing-guide
+[contributing-guide]: https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data
+[default template]: https://github.com/exercism/ruby/blob/master/lib/generator/test_template.erb
+[exercism ruby]: https://github.com/exercism/ruby
+[#generated-test-suites]: #generated-test-suites
+[helper]: https://github.com/exercism/ruby/blob/master/lib/generator/exercise_case.rb
+[#implementing-a-generator]: #implementing-a-generator
+[port exercise]: https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md
+[problem-specifications]: https://github.com/exercism/problem-specifications
 [problem-specifications]: https://github.com/exercism/problem-specifications
+[#pull-requests]: #pull-requests
+[#readmes]: #readmes
+[rebasing documentation]: https://help.github.com/articles/about-git-rebase/
+[#regenerating-a-test-suite]: #regenerating-a-test-suite
+[#regenerating-a-test-suite]: #regenerating-a-test-suite
+[#ruby-icon]: #ruby-icon
+[#running-the-tests]: #running-the-tests
+[rubocop]: http://batsov.com/rubocop/
+[ruby-gitter-badge]: https://badges.gitter.im/exercism/ruby.svg
+[ruby-gitter-channel]: https://gitter.im/exercism/ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+[#setup]: #setup
+[#style-guide]: #style-guide
+[synchronize]: https://help.github.com/articles/syncing-a-fork/
+[tests-workflow-badage]: https://github.com/exercism/ruby/workflows/Exercise%20tests/badge.svg
+[tests-workflow]: https://github.com/exercism/ruby/actions?query=workflow%3A%22Exercise+tests%22+branch%3Amaster
```

Comment:
There was a typo in the first link. I've also changed to use the badge links that GitHub suggests (the second like was also using the deprecated `master` branch)

```suggestion
[tests-workflow-badge]: https://github.com/exercism/ruby/actions/workflows/exercise-tests.yml/badge.svg
[tests-workflow]: https://github.com/exercism/ruby/actions/workflows/exercise-tests.yml
```